package.json

```
@@ -3,11 +3,11 @@
   "version": "2.4.0",
   "description": "A Fastify plugin to provide access to URI components",
   "main": "plugin.js",
-  "typings": "fastify-url-data.d.ts",
+  "types": "fastify-url-data.d.ts",
   "scripts": {
     "test": "npm run test-ts && tap 'test/**/*.test.js'",
     "test-ci": "tap --cov 'test/**/*.test.js' && npm run test-ts",
```

travis was executing the lint check also

```suggestion
    "test-ci": "standard && tap --cov 'test/**/*.test.js' && npm run test-ts",
```